wallet/src/de/schildbach/wallet/ui/DashPayUserActivityViewModel.kt

```diff
@@ -45,9 +47,13 @@ class DashPayUserActivityViewModel(application: Application) : AndroidViewModel(
 
     val userLiveData by lazy {
         liveData(Dispatchers.IO) {
-            userData = platformRepo.getUser(userData.username).first()
-            sendContactRequestState
-            emit(userData)
+            try {
+                userData = platformRepo.getUser(userData.username).first()
+                sendContactRequestState
+                emit(Resource.success(userData))
+            } catch (ex: Exception) {
+                emit(Resource.error("Failed to load Profile", null))
+            }
```

Comment:
In the alpha program, there were many crashes that originated from `PlatformRepo.searchUsernames`.  This code here will at least prevent the exception from crashing the app.  Another PR that I have yet to submit will attempt to fix the root cause.